understand/observed-size-spectra.qmd

```diff
@@ -319,15 +318,15 @@ Fitting a same linear model to the binned biomass density data now gives
 (model <- lm(log(Biomass_dens) ~ log(mid_points[bin]), data = binned_biomass))
 ```
 
-The slope is `r signif(model$coefficients[[2]], 4)` for the biomass density whereas it was `r exponent` for the number density. This makes perfect sense, because if we denote the biomass density by $B(w)$ then $B(w) = w N(w)$. Therefore if numbers at weight scale with the exponent $\lambda$ as $N(w) \propto w^{-\lambda}$ then biomass at weight will scale as $B(w)\propto w^{1-\lambda}$.
+The slope is `r signif(model$coefficients[[2]], 4)` for the biomass density whereas it was `r exponent` for the number density. This makes perfect sense, because if we denote the biomass density by $B(w)$ then $B(w) = w N(w)$. Therefore if the number density scales with the exponent $\lambda$ as $N(w) \propto w^{-\lambda}$ then the biomass density will scale as $B(w)\propto w^{1-\lambda}$.
 
 ## A few final points before the summary: 
 
 ### Densities in log weight
 
 There is perhaps something a bit strange about the way we visualised the size spectrum via the number density or the biomass density plotted on logarithmic axes.
 
-Above we calculated densities by dividing the total number or total biomass in each bin by the width that the bin has on the linear weight ($w$) axis. That is fine. Then we decided to use a logarithmic axis that showed $\log(w)$. But then would it not have been more natural to calculate the densities by dividing by the width of the bin on that logarithmic axis? **ASTA - I can't follow this and we need some example here. Log_bin_width is just 1 1 1 1 1 1. So what's the point of dividing by it?**
```

Comment:
This we need to discuss. Yes, in our example we happened to be using bin width on the logarithmic axis equal to 1 so that dividing does not change anything. But we could have chosen other bin widths.

Reply:
You say that we need an example. Should we use the example from exercise 1 where the bin width on the logarithmic scale was decreased to 1/2?